pvlib/iotools/sodapro.py

```diff
@@ -0,0 +1,361 @@
+    'Clear sky BHI': 'bhi_clear',
+    'Clear sky DHI': 'dhi_clear',
+    'Clear sky BNI': 'dni_clear',
+    'GHI': 'ghi',
+    'BHI': 'bhi',
+    'DHI': 'dhi',
+    'BNI': 'dni',
+    'sza': 'solar_zenith',
+}
+
+# Dictionary mapping time steps to CAMS time step format
+TIME_STEPS_MAP = {'1min': 'PT01M', '15min': 'PT15M', '1h': 'PT01H',
+                  '1d': 'P01D', '1M': 'P01M'}
+
+TIME_STEPS_IN_HOURS = {'1min': 1/60, '15min': 15/60, '1h': 1, '1d': 24}
+
+SUMMATION_PERIOD_TO_TIME_STEP = {'0 year 0 month 0 day 0 h 1 min 0 s': '1min',
+                                 '0 year 0 month 0 day 0 h 15 min 0 s': '15min',  # noqa
+                                 '0 year 0 month 0 day 1 h 0 min 0 s': '1h',
+                                 '0 year 0 month 1 day 0 h 0 min 0 s': '1d',
+                                 '0 year 1 month 0 day 0 h 0 min 0 s': '1M'}
+
+
+def get_cams(start_date, end_date, latitude, longitude, email,
+             identifier='mcclear', altitude=None, time_step='1h',
+             time_ref='UT', verbose=False, integrated=False, label=None,
+             map_variables=True, server='www.soda-is.com'):
+    """
+    Retrieve time-series of radiation and/or clear-sky global, beam, and
+    diffuse radiation from CAMS [1]_, [2]_ using the WGET service [3]_.
+
+    Time coverage: 2004-01-01 to two days ago
+
+    Access: free, but requires registration, see [1]_
+
+    Requests: max. 100 per day
+
+    Geographical coverage: Wordwide for CAMS McClear and -66° to 66° in both
+    latitude and longitude for CAMS Radiation
+
+
+    Parameters
+    ----------
+    start_date: datetime like
+        First day of the requested period
+    end_date: datetime like
+        Last day of the requested period
+    latitude: float
+        in decimal degrees, between -90 and 90, north is positive (ISO 19115)
+    longitude : float
+        in decimal degrees, between -180 and 180, east is positive (ISO 19115)
+    email: str
+        Email address linked to a SoDa account
+    identifier: {'mcclear', 'cams_radiation'}
+        Specify whether to retrieve CAMS Radiation or McClear parameters
+    altitude: float, default: None
+        Altitude in meters. If None, then the altitude is determined from the
+        NASA SRTM database
+    time_step: str, {'1min', '15min', '1h', '1d', '1M'}, default: '1h'
+        Time step of the time series, either 1 minute, 15 minute, hourly,
+        daily, or monthly.
+    time_reference: str, {'UT', 'TST'}, default: 'UT'
+        'UT' (universal time) or 'TST' (True Solar Time)
+    verbose: boolean, default: False
+        Verbose mode outputs additional parameters (aerosols). Only available
+        for 1 minute and universal time. See [1] for parameter description.
+    integrated: boolean, default False
+        Whether to return radiation parameters as integrated values (Wh/m^2)
+        or as average irradiance values (W/m^2) (pvlib preferred units)
+    label: {‘right’, ‘left’}, default: None
```

Comment:
nitpick, use simple quotes instead of fancy ones (here and in the lines below).  The docstrings of other functions have it too

Reply:
Always good to be consistent. I suppose I copied it from another function. Thanks!